### ISSUE TITLE
[ngAria] Should not include aria-required on radiobutton or checkboxes

### ISSUE THREAD

**Issue by monkier** [angular/angular.js#15572]
Currently if `ng-required` exists on an input of type radio or checkbox, `aria-required` is falsely included.

As per the [WAI-ARIA spec](https://www.w3.org/TR/wai-aria/states_and_properties#aria-required), aria-required is not supported for these types of input and [aXe-core](https://github.com/dequelabs/axe-core/blob/master/doc/API.md#options-parameter) tests/audit and [Google Accessibility Dev Tools audits fail](https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_aria_10).

ngAria Version: 1.3.20

**Comment by gkalpak** [comment]
This has already been fixed with #13483 in 1.5.x/1.6.x.
